deploy/cr.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
 #    - delete-pxc-pvc
 spec:
   secretsName: my-cluster-secrets
+#  usersSecrets:
+#  - "secret-for-users"
```

Comment:
we need to add example into `deploy/app-users.yaml`

Reply:
added https://github.com/percona/percona-xtradb-cluster-operator/pull/342/commits/a457d270418b7ddce6f9b7ec3f859fda1bd21b36